config/initializers/devise.rb

```diff
@@ -0,0 +1,336 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # config.secret_key = 'ddaae8e85b0b01ad44ed021fb9c8b21b8e616bfb41613eefc7df121058cc67347632159785f64502bf89e4e5b2290273b66d0dc58bc026a458bc16e0bcab5c49'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'RADenquiries@moneyadviceservice.org.uk'
```

Comment:
No biggy, but I'm wondering if this should be in config. Especially as this is a public repo.